### ISSUE TITLE
Profile fields lost in invite

### ISSUE THREAD

**Issue by JohnAJ** [justingit/dada-mail#1121]
When inviting a single address to a list, and filling out all profile fields, when (if) the invite is accepted the profile fields seem to be lost.

Haven't tests with multiple addresses / CSV etc.

Dadamail Pro 11.19.0 2022-08-20

**Comment by justingit** [comment]
It is not entirely clear (and that lack of clarity is bug-worthy) but the list invite would take in the values set in the form shown to the invitee (not what you fill out yourself). If they don't fill anything out, there will be non profile field values.

**Comment by JohnAJ** [comment]
OK... then why have profile fields in admin when sending the invite if they are ignored? Either don't show them, or make any values set with the invite pre-fill the form when the invitation is accepted?

**Comment by justingit** [comment]
> OK... then why have profile fields in admin when sending the invite if they are ignored?

Yeah I agree - and what I mean by "that's bugworthy" 

> or make any values set with the invite pre-fill the form when the invitation is accepted?

GDPR

> Or... are any hidden ones (prefixed with _) kept? (haven't tested this!)

I do not believe so.

**Comment by JohnAJ** [comment]
:D fair enough, kinda what I figured.

GDPR sucks (and yes, I'm in Europe!).

Thanks :)

**Comment by justingit** [comment]
Yeah, I wish it was different - even jut having the list invite feature may be a bit of a grey area as you're technically sending a mass email out to a unconfirmed address - you're just not doing it again .

**Comment by JohnAJ** [comment]
Just to expand on this (and maybe turn it into a feature request rather than a bug report)...

It' would be handy to have any _hidden_ profile fields survive the invitation process. I have one - _via - that's usually populated with something like "widget_show_2022" etc. etc. for trade show business card collections etc... and sending a partial invite to where _via="widget_show_2022" to send to only these subscribers (if they accepted the invite) would be good.

> Yeah, I wish it was different - even jut having the list invite feature may be a bit of a grey area as you're technically sending a mass email out to a unconfirmed address - you're just not doing it again .

OK, but GDPR says you can send emails to recipients you've collected info on for "legitimate interests" in the normal course of business, i.e. they visited your trade show stand and handed you their business card. So invites are OK, as long as they're genuine and not abused.
